### PR TITLE
Healthcheck example

### LINKED ISSUE #176: Add health check to example
> 
> > Also, does the following o/p from the test certify that the job is running and is in healthy state? It says created so am not sure about the running state.
> > `nomad_job.countdash: Creating... nomad_job.countdash: Still creating... [10s elapsed] nomad_job.countdash: Still creating... [20s elapsed] nomad_job.countdash: Still creating... [30s elapsed] nomad_job.countdash: Creation complete after 35s [id=countdash]`
> 
> This indicates a job is running and healthy, correct. This task will **not** finish unless the task is healthy in nomad.
This is only true if the job has a functioning **health check**. This example currently does not.

_Originally posted by @fredrikhgrelland in https://github.com/fredrikhgrelland/vagrant-hashistack/issues/170#issuecomment-672641291_

### PR DESCRIPTION
**Pull request by pdmthorsrud**
Closes #176

### CODE REVIEW

**Comment by pdmthorsrud** [comment]
A question about etiquete: is this the correct use of `CHANGELOG.md`?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [0.3.1 UNRELEASED]
+
+### Added
+
+- Added healthchecks to the [countdash-job](template/example/nomad/countdash.hcl).
```
Comment (fredrikhgrelland) [inline CHANGELOG.md:7]: This should be added to 0.3.0 UNRELEASED.

You should only add a new section if there is not already a UNRELEASED one.

Comment (pdmthorsrud) [reply CHANGELOG.md]: Thank you. Fixed!

**Comment by pdmthorsrud** [comment]
> While we are at it. These changes should be rebased into one single commit with a meaningful commit message.

Fair enough, it's done. :)

**Pull request merged**